.travis.yml

```diff
@@ -2,6 +2,8 @@ language: ruby
 sudo: false
 rvm:
 - ruby-head
+- 2.3
```

Comment:
I believe this should be "2.3.0", the RVM in Travis CI isn't finding a Ruby for the alias "2.3"